app/store/index.ts

```diff
@@ -28,6 +28,7 @@ const actions = {};
 
 const plugins: any[] = [];
 
+let mutationId = 1;
```

Comment:
is there a reason we want to store IDs like this instead of something like generating a uuid?

Reply:
Is there a reason we want to use uuid instead of much faster integer incrementation? This id is needed only per application life, we don't store it in configs.

Reply:
no reason specifically was just wondering since it was the first time i saw this pattern